frontend/webpage/src/components/RoomCount.vue

```diff
@@ -0,0 +1,63 @@
+<template >
+  <div class="container">
+    <div>Room name: {{ roomName }}</div>
+    <div>Room count: {{ roomCount }}</div>
+    <div>Last updated: {{ lastUpdated }}</div>
+  </div>
+</template>
+<script>
+export default {
+  name: "RoomCount",
+  props: {
+    roomInfo: Object,
+  },
```

Comment:
Add this method to format the date nicely
```suggestion
  },
    methods: {
        addZeroToDate(date) {
            return date < 10 ? "0" + date : date;
        },
    },
```

Reply:
Fixed.